app/views/docs/permissions.phtml

```diff
@@ -94,6 +94,10 @@
         <td>Role.team([TEAM_ID], [ROLE])</td>
         <td>Grants access to any member who possesses a specific role in a team. To gain access to this permission, the user must be a member of the specific team and have the given role assigned to them. Team roles can be assigned when inviting a user to become a team member.</td>
     </tr>
+    <tr>
+        <td>Role.member([MEMBERSHIP_ID])</td>
+        <td>Grants access to a specific member of a team. When the member is removed from the team, the permission is no longer valid.</td>
```

Comment:
> the permission is no longer valid

Sounds like the permission itself is invalid, like a bug or error. How about something like:

```suggestion
        <td>Grants access to a specific member of a team. When the member is removed from the team, they will no longer have access.</td>
```